records/time.go

```diff
@@ -0,0 +1,14 @@
+package records
+
+import "time"
+
+func Time(t int64) time.Time {
+	return time.Unix(t/1000, (t%1000)*int64(time.Millisecond)).UTC()
+}
+
+func Timestamp(t time.Time) int64 {
+	if t.IsZero() {
+		return 0
+	}
+	return t.UnixNano() / int64(time.Millisecond)
```

Comment:
```suggestion
	return t.UnixMilli()
```

Reply:
`UnixMilli` was added in 1.17, kafka-go still maintains compatibility with stable Go versions, so 1.15 at this time (and 1.16 when 1.18 is released), so unfortunately I don't think we can use this API yet.

Reply:
Ah k good to know.